### DISCUSSION TITLE
Unread state stuck in GitHub iOS notifications screen

### DISCUSSION THREAD

**Discussion by djc** [community/community#12382] [Mobile] [answered]
Since yesterday, I'm seeing that notifications in the GitHub iOS notifications don't come out of unread state after tapping in to read them. As far as I can see, the content fully loads, but when I go back to the Notifications screen the blue dot is still there.

Is it possible there's a regression here?

**Comment by canuckjacq** [comment, marked as answer]
Hi @djc - thanks for the report. It looks as though there's a fix already merged for this, so it's just a matter of waiting for the next release.

**Reply by djc** [reply]
That's great news, looking forward to it!
